### PR TITLE
Added client secret in helm values.yaml

### PR DESCRIPTION
**Pull request by evalsocket**
Flyte Propeller auth secret is hardcoded. This doesn't allow helm users to override it through values. Just added client secret in values.yaml

Fix https://github.com/flyteorg/flyte/issues/1808

### CODE REVIEW

**Comment by evalsocket** [comment]
@EngHabu do we want kustomize changes ?

File: charts/flyte-core/values-eks.yaml [hunk 1]
```diff
@@ -111,6 +111,7 @@ flytepropeller:
       ephemeral-storage: 1Gi
       memory: 2Gi
   cacheSizeMbs: 1024
+  client_secret: foobar
```
Comment (EngHabu) [inline charts/flyte-core/values-eks.yaml:114]: We only need to set it in values file, right? why do we also need to do that in vlaues-eks?

**Review by EngHabu — COMMENTED** [review]
LGTM... one question

File: charts/flyte-core/values-eks.yaml (hunk 1 above)
Comment (evalsocket) [reply charts/flyte-core/values-eks.yaml:114]: yes

**Review by EngHabu — APPROVED** [review]

**Pull request merged**